src/platform/OpenThread/OpenThreadUtils.cpp

```diff
@@ -100,7 +100,7 @@ void LogOpenThreadStateChange(otInstance * otInst, uint32_t flags)
 
     static char strBuf[64];
 
-    ChipLogDetail(DeviceLayer, "OpenThread State Changed (Flags: 0x%08x)", flags);
+    ChipLogDetail(DeviceLayer, "OpenThread State Changed (Flags: 0x%08" PRIu32 ")", flags);
```

Comment:
Given the 0x - should this flag print be a PRIx32 iso PRIu32 ?

Reply:
Yes, good catch!